Could not find a declaration file for module 'inquirer'.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

```
node_modules/nest-commander/src/inquirer.service.d.ts:2:42 - error TS7016: Could not find a declaration file for module 'inquirer'. '/usr/src/app/node_modules/nest-commander/node_modules/inquirer/lib/inquirer.js' implicitly has an 'any' type.    
Try `npm i --save-dev @types/inquirer` if it exists or add a new declaration (.d.ts) file containing `declare module 'inquirer';`
```

### Minimum reproduction code

1. install nest-commander
2. create a command with it
3. run app

### Expected behavior

No error is shown.

### Package

- [X] <code>nest-commander</code>
- [ ] <code>nest-commander-schematics</code>
- [ ] <code>nest-commander-testing</code>

### Package version

2.3.4

### Node.js version

17.2.0

### In which operating systems have you tested?

- [ ] macOS
- [ ] Windows
- [X] Linux

### Other

_No response_

--- a/.changeset/brown-mice-heal.md
+++ b/.changeset/brown-mice-heal.md
@@ -1,0 +1,5 @@
+---
+'nest-commander': patch
+---
+
+Update peerDependencies for nest-commander to include @types/inquirer

--- a/apps/docs/docs/features/inquirer.md
+++ b/apps/docs/docs/features/inquirer.md
@@ -5,6 +5,12 @@ sidebar_title: Inquirer
 
 So now you've got a great CLI application, but what happens when someone runs the command and forgets to pass in a parameter? Well, right now it's going to throw an error back out and the process will fail. But what if we want to stop the execution and ask the user for input instead of failing outright? That's where the `InquirerService` comes in. With the `InquirerService` we're able to define a set of questions to ask and get the inputs back, then continue on with the normal execution.
 
+:::info
+
+If using the `InquirerService` make sure to install `@types/inquirer` as a `devDependency` to ensure typescript doesn't encounter any type errors
+
+:::
+
 ## Creating a QuestionSet
 
 To `nest-commander` a `QuestionSet`, as it sounds, is a set of related questions that should all be asked together. To create a question set, simply create a class and decorate it with the `@QuestionSet()` decorator. Provide the decorator a name via the options as well, as this name will be used by the `InquirerService` to know which question set to ask. Let's expand on the previous task runner. Say now maybe the user forgets to pass in a command, but we still want to let them define it at runtime. The first thing we need to do is change the `<task>` to `[task]` to make Commander understand that the argument is optional, even though we know it's really not. After that we need to define the question set and question(s) to ask the user. We'll call the question set `task` and define it as follows:
@@ -40,8 +46,9 @@ export class TaskRunner implements CommandRunner {
   ): Promise<void> {
     let task = inputs[0];
     if (!task) {
-      task = (await this.inquirer.ask<{ task: string }>('task', undefined))
-        .task;
+      task = (
+        await this.inquirer.ask<{ task: string }>('task', undefined)
+      ).task;
     }
     const echo = spawn(task, {
       shell: options.shell ?? userInfo().shell

--- a/packages/nest-commander/package.json
+++ b/packages/nest-commander/package.json
@@ -29,6 +29,7 @@
   },
   "peerDependencies": {
     "@nestjs/common": "^8.0.0",
-    "@nestjs/core": "^8.0.0"
+    "@nestjs/core": "^8.0.0",
+    "@types/inquirer": "^8.1.3"
   }
 }

